client/src/components/controls/EventChart.vue

```diff
@@ -146,23 +147,40 @@ export default {
         return;
       }
       canvas.width = this.clientWidth;
-      canvas.height = bars.slice(-1)[0].top + 10;
+      canvas.height = bars.slice(-1)[0].top + 15;
       bars.forEach((bar) => {
-        let padding = 0;
+        const typeColor = bar.color ? bar.color : '#4c9ac2';
         if (bar.selected) {
-          ctx.fillStyle = this.$vuetify.theme.themes.dark.accent;
-
-          ctx.fillRect(bar.left, bar.top, bar.width, 10);
-          padding = 2;
+          ctx.fillStyle = 'white';
+          bar.markers
+            .map((n) => this.x(n))
+            .slice(0, bar.markers.length - 1)
+            .forEach((m, i) => {
+              const current = bar.markers[i];
+              const next = bar.markers[i + 1];
+              const width = (current + 1 === next) ? 2 : 5;
```

Comment:
Depending on the style we choose for the markers, this needs to be modified to be a ratio of the entire width instead of a hard-coded number.  While the y-axis scrolls the x-axis will scale based on resizing so 5 at smaller sizes could compose of several frames instead of one.

Reply:
Yeah, I tried to figure out what was going on to do that and couldn't, and decided not to spend more time on it.